tools/dockerfile/interoptest/grpc_interop_pythonasyncio/build_interop.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Seems identical to https://github.com/grpc/grpc/blob/master/tools/dockerfile/interoptest/grpc_interop_python/build_interop.sh 

Consider introducing build_interop.sh.template and generate both files to avoid the need to maintain these two files separately?

Reply:
Good idea! Created a new include file, and two template files.